### DISCUSSION TITLE
MCP Clients Repo

### DISCUSSION THREAD

**Discussion by bukit-kronik** [modelcontextprotocol/.github#74] [Ideas]
We need more MCP clients than Claude to balance this 'open source'.

**Comment by jspahrsummers** [comment]
You can find a list of clients here: https://modelcontextprotocol.io/clients

These will continue to exist in separate repositories, and may not even themselves be open source, due to clients being much more heavyweight than servers—and potentially containing commercial IP.

However, do have interest in building an example MCP open source client in future!
